apps/gp2-frontend/src/tags/ResultList.tsx

```diff
@@ -39,7 +36,6 @@ const ResultList: React.FC<ResultListProps> = ({ filters = new Set() }) => {
       numberOfPages={numberOfPages}
       currentPageIndex={currentPage}
       renderPageHref={renderPageHref}
-      isAdministrator={isAdministrator}
```

Comment:
Are you sure we want to remove this? Just asking because it's not written in the ticket that we want to remove the condition to enable only admins to export outputs

Reply:
I removed the isAdministrator prop because it wasn't controlling access to export functionality but rather controlling the ui displayed (whether the export functionality is rendered as a link or button). Currently output exports can be done whether the user is an administrator or not. Confirming with Tiffany if this shouldn't be the case.

Reply:
Confirmed that all users can export outputs